decidim-system/app/views/decidim/system/admins/index.html.erb

```diff
@@ -25,7 +25,9 @@
         </td>
         <td class="actions">
           <%= link_to t("actions.edit", scope: "decidim.system"), [:edit, admin] %>
-          <%= link_to t("actions.destroy", scope: "decidim.system"), admin, method: :delete, class: "small alert button", data: { confirm: t("actions.confirm_destroy", scope: "decidim.system") } %>
+          <% if !current_admin?(admin) %>
```

Comment:
is there any reason why don't you use unless ?

Reply:
A very good question! Now that I see it I'm wondering the same, just at the start of my career so these weird ways of doing things come up :smile: I can change it :+1:

Reply:
Thank you! :) It is changed :+1: